src/ignitus-Shared/ignitus-DesignSystem/ignitus-Molecules/ignitus-Overlay/Components/index.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { mount, render } from 'enzyme';
+import emotionSerializer, { matchers } from 'jest-emotion';
+import Overlay from './index';
+
+const props = {
+  title: 'Title',
+  primaryText: 'primary text',
+  secondaryText: 'secondary text',
+  buttonText: 'button text',
+  secondary: true,
```

Comment:
Remove this one, there is no definition of ```secondary``` in ```Overlay``` component.

Reply:
https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Molecules/ignitus-Overlay/types.ts
Isn't it given in this file?

Reply:
It is not used in ```Overlay``` component.

Reply:
Removed